site/docs/reference/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Reference",
+  "position": 7,
+  "link": {
+    "type": "generated-index",
+    "description": "Reference to OpenVINO GenAI."
```

Comment:
As far as I remember `Reference` was added as an example. If you don't know what could go here, it's time to remove it.

Reply:
Removed